configure.ac

```
@@ -1142,6 +1141,18 @@ else
   AC_MSG_RESULT([no])
 fi
 
+# When compiled natively on MacOS, we need to specify --flat to avoid producing a dSYM bundle
+# When cross-compiled on linux, we're using a different version of the tool that only supports flat symbol files
+AC_MSG_CHECKING([whether dsymutil needs --flat])
+if test x$DSYMUTIL != x && ($DSYMUTIL --help | grep -q \\--flat); then
```

It's  `-flat` in help text (though you can use `--flat` when running `dsymutil`)
```suggestion
if test x$DSYMUTIL != x && ($DSYMUTIL --help | grep -q \\-flat); then
```